csi/node.go

```diff
@@ -159,9 +159,8 @@ func (s *OsdCsiServer) NodeUnpublishVolume(
 	// Get volume information
 	vol, err := util.VolumeFromName(s.driver, req.GetVolumeId())
 	if err != nil {
-		return nil, status.Errorf(codes.NotFound, "Volume id %s not found: %s",
-			req.GetVolumeId(),
-			err.Error())
+		logrus.Infof("Volume %s was deleted or cannot be found: %s", req.GetVolumeId(), err.Error())
```

Comment:
instead of ignoring all errors, let's only ignore volume not found errors.

For all other errors, we should fail as before.

Reply:
got it - will check for kvdb.ErrNotFound